test/integration/h2_capture_fuzz.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package test.integration;
+
+enum H2TestFrame {
+  // These values map to the frame creation methods in:
+  // test/common/http/http2/http2_frame.h
+  PING = 0;
```

Comment:
How come `PING` is all caps but the rest are camel case?

Reply:
Fixed